uk.co.compucorp.civicrm.hrcore/templates/CRM/HRCore/Page/Error404Checker.tpl

```diff
@@ -0,0 +1,8 @@
+<p>
+  The page you are looking for can't be found.  Maybe you could try:
+</p>
+<ul>
+  {foreach from=$urls item="currentUrl" key="URLLabel"}

```

Comment:
inconsistency between Url and URL. Stick to only one. I would say to stick to Url, because then you can write the key as `$urlLabel`